extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -403,6 +405,13 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		}
 	}
 
+	public async openRemoteBook(): Promise<void> {
```

Comment:
This doesn't seem to need to be a part of the BookTreeViewProvider - it doesn't use anything from the provider directly. Probably just move this to extension.ts directly